### PR TITLE
Create hemu-codes.md

### PR DESCRIPTION
**Pull request by hemu-codes**
### Description
Created a profile of myself - which includes a short description and some other background information. 

### Raw.Githack preview link:

https://raw.githack.com/hemu-codes/hemu-codes.github.io/add-hemucodes-profile/#!pages/vi/profiles/hemu-codes.md

### CODE REVIEW

**Comment by jeevast** [comment]
@hemu-codes the link you have given here isn't the exact one, please check again with the GitHub and Markdown steps.

**Comment by hemu-codes** [comment]
Could you please look over it again? I made a couple of changes. Let me know if I need to fix anything else. Thank you.

File: pages/vi/profiles/hemu-codes.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+* OS: Mac Version 10.14.5
+-------------------------------------
+
+## Description: 
+Hello, I am Hemantha - a sophomore attending the University of Washington Bothell. 
+I am pursuing a degree in Mathematics and Computer Science. Some of the extracurricular activities I am part of include 
+Undergraduate Research and officer for some of the organizations at my school. I am really excited to be part of this team
+and looking forward to working with you all!!
+
+Linkedin: www.linkedin.com/in/hemantha-akkaraju
```
Comment (lmmrssa) [inline pages/vi/profiles/hemu-codes.md]: this is a link so can you add link markdown here

Comment (hemu-codes) [reply pages/vi/profiles/hemu-codes.md]: @lmmrssa made the suggested change. Could you please look over it again. Thank you!

File: pages/vi/profiles/hemu-codes.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* OS: Mac Version 10.14.5
+-------------------------------------
+
+## Description: 
+Hello, I am Hemantha - a sophomore attending the University of Washington Bothell. 
+I am pursuing a degree in Mathematics and Computer Science. Some of the extracurricular activities I am part of include 
+Undergraduate Research and officer for some of the organizations at my school. I am really excited to be part of this team
+and looking forward to working with you all!!
+
+[Linkedin](www.linkedin.com/in/hemantha-akkaraju)
```
Comment (lmmrssa) [inline pages/vi/profiles/hemu-codes.md]: link does not work

Comment (hemu-codes) [reply pages/vi/profiles/hemu-codes.md]: Fixed the link! Could you please check if it works now? Thanks again!

File: pages/vi/profiles/hemu-codes.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ I am pursuing a degree in Mathematics and Computer Science. Some of the extracur
 Undergraduate Research and officer for some of the organizations at my school. I am really excited to be part of this team
 and looking forward to working with you all!!
 
-[Linkedin](www.linkedin.com/in/hemantha-akkaraju)
+[Linkedin](https://www.linkedin.com/feed/)
```
Comment (lmmrssa) [inline pages/vi/profiles/hemu-codes.md]: why are you taking to feed it was supposed to be your linkedin page. There is no meaning of opening linkedin default page on someone's profile.

Comment (hemu-codes) [reply pages/vi/profiles/hemu-codes.md]: My individual profile link wasn't working earlier so I changed it to another one. But I changed it back to my profile link (which is working for me) but I'm not sure if you are able to access it.

**Pull request merged**